src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,7 +2800,7 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
Remoting loopback tests.Can execute command in a disconnected session
```powershell
Objects cannot be added to a closed buffer. Make sure the buffer is open for Add and Insert operations to succeed.
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersRemoting\HelpersRemoting.psm1: line 225
225:     Invoke-Command @parameters

```

Reply:
@PoshChan please retry windows

Reply:
@JamesWTruher if you're able would love any insight you can offer here. This has happened almost 50% of the time in these ci runs for this PR. Only on Windows. I suspect moving the dispose() executions up has created a race condition or something in a remoting context?

Reply:
Not sure how to go about troubleshooting something like this...